src/EnduranceTests/TestUtils/BlobNameBuilder.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace NServiceBus.AzureServiceBus.EnduranceTests.TestUtils
+{
+    using System;
+    using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.Blob;
+    using NLog;
+
+    public static class BlobNameBuilder
+    {
+        public static string Build(LogEventInfo logEvent)
+        {
+            return $"{logEvent.TimeStamp.Ticks}_{logEvent.Exception.GetType().Name}";
+        }
+
+        public static string BuildContainerName(LogEventInfo logEvent)
+        {
+            var name = logEvent.LoggerName.Replace(".", "-").Replace(" ", "").ToLower();
+            return name.Length > 63 ? name.Substring(0, 63) : name;
+        }
+
+        public static string Build(CloudStorageAccount storageAccount, LogEventInfo logEvent)
```

Comment:
Both `Build` methods look like an overload, but are they building the same thing? Hard to figure from the name itself.